crates/sui-indexer/migrations/pg/2024-09-19-011238_raw_checkpoints/up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE raw_checkpoints
+(
+    sequence_number                     BIGINT       PRIMARY KEY,
+    certified_checkpoint                BYTEA        NOT NULL,
```

Comment:
One thought: Would there be any value in storing the checkpoint summary and the validator quorum signatures separate in different columns?

I ask this because when we introduce a new table for Transactions, I think we really do want to separate out the TransactionData payload from the UserSignature. One to address the base serialization format with Intent and two because the signature data can be retrieved from the checkpoint so we can save some space.

Reply:
> because the signature data can be retrieved from the checkpoint so we can save some space.

For that you need the user signatures from checkpoint_contents, not the quorum signatures, or did I misunderstood?

Reply:
Sorry these are two separate thing. Yes for UserSignatures you'd get that from the contents. The question wrt the quorum signatures is just: because we should do this separation from Transactions, should we do the same here for quorum signatures from the checkpoint summary.

Reply:
Maybe? They are not quite the same though, one is signature from user, while here are signatures from validators.